library/functions.lisp

```diff
@@ -82,6 +82,7 @@
   ;; considered to be "saturated".
   (declare compose ((:b -> :c) -> (:a -> :b) -> (:a -> :c)))
   (define (compose f g)
+    "Consumes two procedures and returns a procedure of one arg that composes F and G."
```

Comment:
I would say something closer to: "produces a function equivalent to applying G then F in succession"

Reply:
I would also put on a subsequent line the identity that
```
((compose f g) x)
```
Behaves like
```
(f (g x))
```